code/modules/projectiles/ammo_datums.dm

```diff
@@ -543,6 +677,24 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	penetration = 15
 	sundering = 2
 
+/datum/ammo/bullet/rifle/standard_dmr/polymer
+	name = "marksman polymer gel round"
+	icon_state = "bullet_pgr"
+	handful_icon_state = "bullet_pgr"
+	hud_state = "hivelo_pgr"
+	hud_state_empty = "hivelo_empty"
+	damage_falloff = 0.5
+	flags_ammo_behavior = AMMO_BALLISTIC
+	accurate_range_min = 0
+	accurate_range = 30
+	max_range = 40
+	damage = 65
+	damage_type = STAMINA
+	scatter = -15
+	penetration = 20
```

Comment:
```suggestion
	hud_state_empty = "hivelo_empty"
	damage = 65
	damage_type = STAMINA
	penetration = 20
```
There's more of these, but there's no need to list all of them.
It doesn't actually make any difference ingame of course, but it makes the code a lot cleaner.